From b656a77864bd322d69522f1f9d922404066e5a7c Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Mon, 21 Oct 2019 09:32:15 +0200
Subject: [PATCH] libtorrent.pc.in: add Libs.Private (#202)

Add Libs.Private: -lz so applications that want to link statically with
libtorrent (such as rtorrent) will know that they must link with -lz

Fixes:
 - http://autobuild.buildroot.org/results/075598e1699c2ac20a4dfbcb5695bbb7343f9a86

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 libtorrent.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libtorrent.pc.in b/libtorrent.pc.in
index cf6612bc..6108f7e9 100644
--- a/libtorrent.pc.in
+++ b/libtorrent.pc.in
@@ -7,4 +7,5 @@ Name: libtorrent
 Description: A BitTorrent library
 Version: @VERSION@
 Libs: -L${libdir} -ltorrent
+Libs.Private: -lz
 Cflags: -I${includedir}
